examples/ContextExample.js

```diff
@@ -138,36 +139,12 @@ const PagedCell = (props, { data, version }) => (
     {...props}
   />);
 
-/**
- * Data type validator
- *
- * Instead of having React.PropTypes.instanceOf(PagedData) the custom PropTypes
- * allows any data structure as long as it provides a `getObjectAt` function.
- *
- * @param {object} props         The list of the props
- * @param {string} propName      The prop that is to be validated
- * @param {string} componentName The name of the component that the prop belongs to
- */
-function PropTypeCtxtData(props, propName, componentName) {
```

Comment:
Since propTypes aren't a FDT specific thing and they clutter these examples, I've moved them to a helper file